debian/control

```diff
@@ -21,11 +23,11 @@ Build-Depends: dpkg-dev (>= 1.14.8), debhelper (>= 9), dh-autoreconf,
  libbullet-dev, libpulse-dev, liblz4-dev, libwebp-dev, libharfbuzz-dev,
  libsystemd-dev, libavahi-client-dev, systemd, libvlc-dev, libraw-dev,
  libpoppler-cpp-dev, libspectre-dev, librsvg2-dev
-Build-Depends-Indep: doxygen, imagemagick
+Build-Depends-Indep: doxygen, ghostscript, imagemagick, texlive-font-utils
```

Comment:
@rvandegrift I definitely had a build failure without those.
You can test yourself by running the build in a clean environment (either with sbuild or pbuilder)